examples/models/population_server.py

```diff
@@ -130,8 +130,8 @@ def create_layout():
     year_select.on_change('value', on_year_change)
     location_select.on_change('value', on_location_change)
 
-    controls = HBox(children=[year_select, location_select])
-    layout = VBox(children=[controls, pyramid(), population()])
+    controls = WidgetBox(children=[year_select, location_select],height=150,width=600)
```

Comment:
@mattpap I am going to open a general PR to collect small improvements to all the examples later today, let's update this there.